### PR TITLE
Switch python2 %s for python3 .format

### PR DESCRIPTION
**Pull request by michaelbadcrumble**
I decided to join the WrapDB team to fix the understaffed issue. First fix is to switch Python2 `%s` for Python3 `.format` method.

### CODE REVIEW

File: mesonwrap/githubdb.py [hunk 1]
```diff
@@ -99,8 +99,9 @@ def _get_asset(org: Organization,
             with requests.get(asset.browser_download_url) as rv:
                 rv.raise_for_status()
                 return rv.content
-    _log.error('Asset not found project=%s branch=%s revision=%d label=%s',
```
Comment (legeana) [inline mesonwrap/githubdb.py:102]: I don't think logging commands should be fixed like that. They are defined using `%` operator and this change makes the code harder to read.

```
error(msg, *args, **kwargs) method of logging.Logger instance
    Log 'msg % args' with severity 'ERROR'.

    To pass exception information, use the keyword argument exc_info with
    a true value, e.g.

    logger.error("Houston, we have a %s", "major problem", exc_info=1)
```

Comment (michaelbadcrumble) [reply mesonwrap/githubdb.py]: Ok, I have reverted that change.

Comment (michaelbadcrumble) [reply mesonwrap/githubdb.py]: Ping @legeana .

File: mesonwrap/wrapcreator.py [hunk 2]
```diff
@@ -31,7 +31,7 @@
 
 # Replace this with proper parameterized callback if this need to be extended.
 _OUT_URL_BASE_DEFAULT = (
-    'https://wrapdb.mesonbuild.com/v1/projects/%s/%s/%d/get_zip')
```
Comment (legeana) [inline mesonwrap/wrapcreator.py:34]: Please also change the call site on the line 87.

Comment (michaelbadcrumble) [reply mesonwrap/wrapcreator.py]: Ok. Changed the call site on line 87. Any other issues?

**Pull request merged**